util/mathutil/math_test.go

```diff
@@ -78,4 +78,28 @@ func TestClamp(t *testing.T) {
 	require.Equal(t, float32(1.0), Clamp(float32(0), 1.0, 3.0))
 	require.Equal(t, 1, Clamp(0, 1, 1))
 	require.Equal(t, 1, Clamp(100, 1, 1))
+	require.Equal(t, "ab", Clamp("aa", "ab", "xy"))
+	require.Equal(t, "xy", Clamp("yy", "ab", "xy"))
+	require.Equal(t, "ab", Clamp("ab", "ab", "ab"))
+}
+
+func BenchmarkMaxMin(b *testing.B) {
```

Comment:
Why add the benchmark test?

Reply:
I tried another implement using `for i := range xs` wandering may be it is faster for stings,
but it is not.
the benchmark is keep for later use(maybe).
do i need to delete it?

Reply:
Yes, please remove it :)

Reply:
> Yes, please remove it :)

deleted